cms/static/js/views/pages/container.js

```diff
@@ -96,6 +99,11 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
                     });
                     this.unitOutlineView.render();
                 }
+                if (this.isLibrarySourced) {
+                    this.selectedLibraryComponents = [];
+                    this.storeSelectedLibraryComponents = [];
```

Comment:
If I got it right this is the array of the ids already stored for the block. In this case, I'm suggesting changing the name: `storeSelectedLibraryComponents` -> `storedSelectedLibraryComponents` (store -> stored)